server/protocols/jmap/src/main/java/org/apache/james/jmap/DownloadServlet.java

```diff
@@ -129,11 +129,12 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
     }
 
     private void addContentDispositionHeader(Optional<String> optionalName, HttpServletResponse resp) {
-        optionalName.ifPresent(name -> resp.addHeader("Content-Disposition", "attachment; filename=\"" + name + "\""));
```

Comment:
Why does file name disapear from Download endpoint ? There might be a good reason, I guess you can document these kind of changes threw commit message....